src/Microsoft.VisualStudio.Threading.Analyzers/Microsoft.VisualStudio.Threading.Analyzers.csproj

```diff
@@ -34,105 +28,50 @@
     <SuppressPseudoWarning Condition="'$(Configuration)' == 'Debug'">true</SuppressPseudoWarning>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="Microsoft.CSharp" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="VSTHRD106UseInvokeAsyncForAsyncEventsAnalyzer.cs" />
-    <Compile Include="VSTHRD200UseAsyncNamingConventionAnalyzer.cs" />
-    <Compile Include="VSTHRD200UseAsyncNamingConventionCodeFix.cs" />
-    <Compile Include="CommonInterest.cs" />
-    <Compile Include="VSTHRD003UseJtfRunAsyncAnalyzer.cs" />
-    <Compile Include="VSTHRD102AvoidJtfRunInNonPublicMembersAnalyzer.cs" />
-    <Compile Include="Namespaces.cs" />
-    <Compile Include="VSTHRD105AvoidImplicitTaskSchedulerCurrentAnalyzer.cs" />
-    <Compile Include="Strings.Designer.cs">
+    <Compile Update="Strings.Designer.cs">
```

Comment:
The new project type includes all files in the project directory automatically, but without metadata. So where metadata is important, we must update the item it includes automatically to add it.